compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Base.hs

```diff
@@ -994,25 +981,11 @@ data TemplateChoice = TemplateChoice
 
 -- | Feature flags for a module.
 data FeatureFlags = FeatureFlags
```

Comment:
Maybe we should drop feature flags from the AST?

Reply:
yeah I considered it, for now it seemed reasonable to keep it in there but I don’t feel strongly about this either way.